lib/rules/no-wait-for-side-effects.ts

```diff
@@ -76,8 +86,32 @@ export default createTestingLibraryRule<Options, MessageIds>({
       }
     }
 
+    function reportImplicitReturnSideEffect(node: TSESTree.CallExpression) {
+      if (!isCallerWaitFor) {
```

Comment:
I think this is checking for the existance of the function, instead of the result of calling it

```suggestion
      if (!isCallerWaitFor(node)) {
```

Reply:
Totally 😅

Reply:
Fixed in 71c97fc